### PR TITLE
fix(vector): Apply various suggested changes from JetBrains markdown and proofreading inspections

### PR DESCRIPTION
**Pull request by spencergilbert**
Signed-off-by: Spencer Gilbert <spencer.gilbert@datadoghq.com>

### CODE REVIEW

File: charts/vector/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: vector
-version: "0.16.3"
+version: "0.16.4"
```
Comment (StephenWakely) [inline charts/vector/Chart.yaml:3]: Why is this necessary?

Comment (StephenWakely) [reply charts/vector/Chart.yaml:3]: Ah.. has this leaked in from the colour codes pr?

Comment (spencergilbert) [reply charts/vector/Chart.yaml:3]: We'd fail the check [here](https://github.com/vectordotdev/helm-charts/actions/runs/3282096145/jobs/5405027736#step:5:29).

The datadog agent does similarly, https://github.com/DataDog/helm-charts/pull/681
